tdrs-backend/gunicorn_start.sh

```diff
@@ -7,4 +7,13 @@ python manage.py migrate
 python manage.py populate_stts
 python manage.py collectstatic --noinput
 echo "Starting Gunicorn"
-exec gunicorn tdpservice.wsgi:application --bind 0.0.0.0:8080 --timeout 10 --workers 3 --log-file=- --log-level $LOGGING_LEVEL
+
+if [[ "$DJANGO_CONFIGURATION" = "Development" ]]; then
+    gunicorn_params="--bind 0.0.0.0:8080 --timeout 10 --workers 3 --reload --log-level $LOGGING_LEVEL"
+else
+    gunicorn_params="--bind 0.0.0.0:8080 --timeout 10 --workers 3 --log-level $LOGGING_LEVEL"
+fi
+
+gunicorn_cmd="gunicorn tdpservice.wsgi:application $gunicorn_params"
+
+exec $gunicorn_cmd
```

Comment:
Thanks to @riatzukiza to running this locally and finding that the previous iteration ran into a gunicorn default of port 8000 due to `exec`'s weird variable handling.